rabbitmq/meta/collectd.yml

```diff
@@ -2,10 +2,11 @@
 local_plugin:
   rabbitmq_info:
     plugin: python
-    execution: local
     template: rabbitmq/files/collectd_rabbitmq_info.conf
-    username: {{ server.admin.name }}
-    password: {{ server.admin.password }}
+    host: {{ server.management.bind.address }}
+    port: {{ server.management.bind.port }}
+    username: guest
+    password: guest
```

Comment:
I've filed https://github.com/tcpcloud/salt-formula-rabbitmq/issues/12 to track this particular issue and I'll revert this part of the change.